tests/cli/test_report.py

```diff
@@ -282,15 +290,25 @@ class args:
         no_header = None
         color = None
 
-    backupfile = BUILD_REPORT + ".bak"
-    shutil.copy2(BUILD_REPORT, backupfile)
+    backupfile_report = tempfile.NamedTemporaryFile()
+    shutil.copy2(BUILD_REPORT, backupfile_report.name)
+
+    backupfile_list_report = tempfile.NamedTemporaryFile()
+    shutil.copy2(BUILDTEST_REPORTS, backupfile_list_report.name)
+
+    report_cmd(args)
 
     # buildtest report clear will raise an error since file doesn't exist
     with pytest.raises(SystemExit):
         report_cmd(args)
 
-    shutil.move(backupfile, BUILD_REPORT)
-    assert BUILD_REPORT
+    assert not is_file(BUILD_REPORT)
+    # move back the backe-up files since report_cmd() function removes the files BUILD_REPORT and BUILDTEST_REPORTS
+    shutil.move(backupfile_report.name, BUILD_REPORT)
+
+    shutil.move(backupfile_list_report.name, BUILDTEST_REPORTS)
+
+    assert is_file(BUILD_REPORT)
```

Comment:
could you also add `assert is_file(BUILDTEST_REPORTS)` just to be consistent since we are moving both files back

Reply:
Once we have this i think this looks good and we can merge

Reply:
@shahzebsiddiqui Done! You can merge this now.